packages/config/src/env.ts

```diff
@@ -0,0 +1,7 @@
+export function env(name: string): string {
+  const value = process.env[name]
+  if (!value) {
+    throw new Error(`Missing required environment variable: ${name}`)
```

Comment:
Nit: would it make sense for this to be a subclass of `Error` with a more distinguished name?
Or to include e.g. `[ValidationEnvError] ` as a prefix to the error message?

Reply:
done